docs/build/ibc-solidity/universal-channel.md

```diff
@@ -26,19 +39,25 @@ For regular IBC packets, both packet sender and receiver with unique IBC ports a
 For Universal Packets, packet sender and receiver still have unique IBC ports for packet routing, but they do not own the channel. Instead, the Universal Channel Middleware contract owns the channel, and it is responsible for handling the channel handshake and authentication.
 One sender can send universal packets over the same universal channel to multiple receivers, and one receiver contract can receive universal packets from multiple senders over the same universal channel too. 
 
-On the sender side, a universal packet is packed into a regular IBC packet and sent over the universal channel. 
+- On the sender side, a universal packet is packed into a regular IBC packet and sent over the universal channel.
+
+- On the receiver side, The Universal Channel Middleware contract unpacks the regular IBC packet, extracts the universal packet, and passes it to the next Middleware, if any, in the middleware stack, until it reaches the final destination specified in `UniversalPacket.destPortAddress` field.
```

Comment:
The UCMW passing extracted packets to the next Middleware may be a bit confusing to users. Is there any way we could simplify this with still being factually accurate. 

On the receiver side, the Universal Channel Middleware contract processes the standard IBC packet to extract the universal packet. It then forwards this universal packet through any other Middleware in the stack, as needed, until it arrives at the final destination specified in the UniversalPacket.destPortAddress field.

Should we add an in section regarding acknowledgment or is it unneccesary?

Reply:
> Should we add an in section regarding acknowledgment or is it unneccesary?

Imo it's not necessary, it should be evident what the flow is given the first part of roundtrip

> On the receiver side, the Universal Channel Middleware contract processes the standard IBC packet to extract the universal packet. It then forwards this universal packet through any other Middleware in the stack, as needed, until it arrives at the final destination specified in the UniversalPacket.destPortAddress field.

Seems like this is mostly rephrasing the paragraph, do you feel this is clear enough?